databases/week2/part3_morequeries.sql

```diff
@@ -0,0 +1,42 @@
+-- Part 3: More queries
```

Comment:
Most queries seem correct. But I think you need to use different tables and values for this part, according to the instructions.